ui/app/app.js

```diff
@@ -96,6 +96,11 @@ export default class App extends Application {
         },
       },
     },
+    sync: {
+      dependencies: {
+        services: ['router', 'store', 'flash-messages'],
```

Comment:
I'm pretty sure we want to include `namespace` here as well

Reply:
I just added a few services here we would be using for sure and anything else can be added as needed. Saves from having some services shared that we might not be accessing.